src/NavBar.js

```diff
@@ -3,14 +3,14 @@ import { View, TouchableOpacity, StyleSheet, Dimensions } from 'react-native';
 import PropTypes from 'prop-types';
 
 // galio components
-import { Block, Text, Icon } from '.';
+import { Block, Text, Icon } from 'galio-framework';
```

Comment:
This is the only difference between this PR (#114) and #116. 

In #116 the import is written correctly as in `... './'`;

Reply:
Yes, you can!!
( A back story why I created these sub PR's,  the current organization I am working with, we create sub PR's for all the small changes for a component, and a master PR which includes all the sub changes, We work like this so that instead of one person having to go through all the changes introduced in a PR, different individuals go through each small changes and then it the master PR is approved )
and I was actually following the same logic here as well!!!